Minimal_API/bin/Debug/net6.0/Minimal_API.deps.json

```diff
@@ -0,0 +1,115 @@
+{
+  "runtimeTarget": {
+    "name": ".NETCoreApp,Version=v6.0",
+    "signature": ""
+  },
+  "compilationOptions": {},
+  "targets": {
+    ".NETCoreApp,Version=v6.0": {
+      "Minimal_API/1.0.0": {
+        "dependencies": {
+          "Swashbuckle.AspNetCore": "6.2.3"
+        },
+        "runtime": {
```

Comment:
Always exclude Bin and Obj folders from PR

Reply:
Deleted bin and obj form the project